staging/src/k8s.io/apiserver/pkg/apis/apiserver/validation/validation_test.go

```diff
@@ -1335,6 +1335,54 @@ func TestValidateClaimMappings(t *testing.T) {
 			structuredAuthnFeatureEnabled: true,
 			want:                          `issuer.claimMappings.extra[0].key: Invalid value: "example.org/Foo": key must be lowercase`,
 		},
+		{
+			name: "extra mapping key prefix is k8.io",
```

Comment:
Add tests that show that `ak8s.io`, `*.ak8s.io`, `bkubernetes.io`, `*.bkubernetes.io`, etc still work.